Resources/views/page_topmenu.html.twig

```diff
@@ -1,23 +1,21 @@
-<div class="navbar-inner">
-    <div class="container">
-
-        <a class="btn btn-navbar" data-action="toggleclass" data-class="nav-collapse" data-target=".main-navi .nav-collapse">{{ "Navigation"|trans }}</a>
+<nav class="navbar navbar-default">
+    <div class="navbar-header">
+        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".navbar-menu" aria-expanded="false" aria-controls="navbar-menu">
+            {{ "Navigation"|trans }}
+        </button>
+    </div>
 
-        <div class="nav-collapse">
-            <div class="nav">
-                {{ knp_menu_render(menu, {'depth': 1, 'firstClass': 'firstli', 'lastClass': 'lastli'}) }}
-            </div>
+    <div class="navbar-collapse collapse navbar-menu navbar-top">
+        <div class="container">
+            {{ knp_menu_render(menu, {'depth': 1, 'firstClass': 'firstli', 'lastClass': 'lastli'}) }}
         </div>
-
     </div>
-</div>
 
-{% if submenu is defined %}
-<div class="navbar-inner sublevel">
-    <div class="container">
-        <div class="nav-collapse">
-            {{ knp_menu_render(submenu, {'firstClass': 'firstli', 'lastClass': 'lastli', 'branch_class': 'nav'}) }}
+    {% if submenu is defined %}
+        <div class="navbar-collapse collapse navbar-menu navbar-sublevel">
+            <div class="container">
+                {{ knp_menu_render(submenu, {'firstClass': 'firstli', 'lastClass': 'lastli', 'branch_class': 'nav'}) }}
```

Comment:
I don't like CSS class names. The could be styled with CSS pseudoselectors like `:first-child`